From 6917e2892290449434f9d22fe358c6ec3663aba3 Mon Sep 17 00:00:00 2001
From: G Edhaya Chandran <edhaya.chandran@arm.com>
Date: Mon, 4 Apr 2022 09:24:13 +0530
Subject: [PATCH] Fix for FWTS build issue

FWTS fails to find zlib.h in new installations of Ubuntu 20.04.
The same has been raised to Canonical. This solution is a temprorary
fix to further the build.
---
 src/lib/include/fwts_fileio.h | 2 +-
 src/lib/src/fwts_fileio.c     | 2 +-
 2 files changed, 2 insertions(+), 2 deletions(-)

diff --git a/src/lib/include/fwts_fileio.h b/src/lib/include/fwts_fileio.h
index 108dae96..21c78ed7 100644
--- a/src/lib/include/fwts_fileio.h
+++ b/src/lib/include/fwts_fileio.h
@@ -21,7 +21,7 @@
 #define __FWTS_FILEIO_H__
 
 #include <stdio.h>
-#include <zlib.h>
+#include </usr/include/zlib.h>
 
 fwts_list* fwts_file_read(FILE *fp);
 fwts_list* fwts_file_open_and_read(const char *file);
diff --git a/src/lib/src/fwts_fileio.c b/src/lib/src/fwts_fileio.c
index 6af6f00f..48e38a1b 100644
--- a/src/lib/src/fwts_fileio.c
+++ b/src/lib/src/fwts_fileio.c
@@ -21,7 +21,7 @@
 #include <stdlib.h>
 #include <string.h>
 #include <unistd.h>
-#include <zlib.h>
+#include </usr/include/zlib.h>
 
 #include "fwts.h"
 
-- 
2.25.1

